assets/scss/components/_needs-scores.scss

```diff
@@ -89,6 +89,14 @@ $white-needs-score-background-colour: #ffffff;
   content: "";
 }
 
+.needsScoreMain {
+  margin-bottom: 0;
+}
+
+.needsScoreSub {
+  margin-bottom: 5px;
```

Comment:
Here if you do `margin-bottom: govuk-spacing(1);` then sizes will auto-update if we decide to change the font size (plus it will be responsive)